app/assets/javascripts/pooled_tubes_from_whole_plates.js

```diff
@@ -103,50 +113,77 @@
 
     $('.plate-box').each(function () {
       $.extend(this, {
-        /*
-          Our plate beds
-        */
         waitLabware: function () {
           this.clearLabware()
-          this.plateContainer().removeClass('good-plate bad-plate scan-plate')
-          this.plateContainer().addClass('wait-plate')
+          this.labwareContainer().removeClass('good-plate bad-plate scan-plate')
+          this.labwareContainer().addClass('wait-plate')
```

Comment:
I don't know the implications, but do these CSS classes need to be changed to indicate they refer to labware rather than specifically plates?

Reply:
Yep will try